src/Internal/Metadata/Cip25/Cip25String.purs

```diff
@@ -142,5 +142,5 @@ toMetadataString str = case toCip25Strings str of
 fromMetadataString :: TransactionMetadatum -> Maybe String
 fromMetadataString datum = do
   fromCip25Strings <$> (Array.singleton <$> fromMetadata datum) <|> do
-    bytes :: Array ByteArray <- fromMetadata datum
-    hush $ decodeUtf8 $ unwrap $ fold bytes
```

Comment:
Actually I might check with @klntsky why we did it this way in the first place

Reply:
No reason I guess

Reply:
Otherwise we can do a similar refactor for `fromDataString` as well:

```purescript
fromDataString :: PlutusData -> Maybe String
fromDataString datum = do
  fromCip25Strings <$> (Array.singleton <$> fromData datum) <|>
    fromCip25Strings <$> fromData datum
```

Reply:
Or even

```
  fromCip25Strings <$> ((Array.singleton <$> fromData datum) <|> fromData datum)
```

Reply:
Would you like me to roll that into this PR?

Reply:
Might as well I think

Reply:
Done